source/_components/device_tracker.tile.markdown

```diff
@@ -37,7 +41,7 @@ device_tracker:
     required: true
     type: string
   monitored_variables:
-    description: the Tile types to monitor; valid values are `TILE` and `PHONE` (default is for all types to be included)
+    description: the Tile types to monitor; valid values are `TILE` and `PHONE` (default is for only 'PHONE' to be monitored)
```

Comment:
See https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/device_tracker/tile.py#L44

Reply:
Then I think the issue isn't the documentation, but a bug in the way the code runs. I'll see if I can find time to test it again, adding the monitored_variables line to my config resolved the issue, though.